app/views/donations/_actions.html.erb

```diff
@@ -1,4 +1,8 @@
-<div class="btn-container">
-  <%= render("donations/confirm", donation: donation) %>
-  <%= render("donations/decline", donation: donation) %>
+<div class="donation-actions">
+  <h3 class="section-label">Would you like to change your donation status?</h3>
```

Comment:
@seanpdoyle I went ahead and moved that markup out, but am getting an error when I try and wrap that block in a check (`undefined method for `donation``)

Do you wanna take a look and make that change? Feel free to merge afterwards.